src/Symfony/Component/Security/Http/ParameterBagUtils.php

```diff
@@ -71,7 +71,7 @@ public static function getParameterBagValue(ParameterBag $parameters, $path)
      *
      * @throws InvalidArgumentException when the given path is malformed
      */
-    public static function getRequestParameterValue(Request $request, $path)
+    public static function getRequestParameterValue(Request $request, string $path)
```

Comment:
The docblock comments for these can be removed. Perhaps the parameter should be renamed to `$key` if the comment is required otherwise, though I think it can be removed without issues the syntax is documented above it